03/db.py

```diff
@@ -0,0 +1,76 @@
+from model import Task
+import sqlite3
+
+MY_TODO_TABLE = 'TODO_LIST'
+
+
+def create_db():
+    con = sqlite3.connect('./todos.db')
+    return con
+
+
+def init_db(con: sqlite3.Connection):
+    cursor = con.cursor()
+    cursor.execute(f'''
+        CREATE TABLE IF NOT EXISTS {MY_TODO_TABLE} (
+            id INTEGER PRIMARY KEY AUTOINCREMENT,
+            title TEXT,
+            content TEXT,
+            done BOOLEAN
+        )''')
+    cursor.close()
+
+
+
+def create_task(con: sqlite3.Connection, title: str, content: str):
+    cursor = con.cursor()
+    cursor.execute(f'''
+        INSERT INTO {MY_TODO_TABLE} (title, content, done)
+        VALUES ('{title}', '{content}', {False})
+    ''')
+    cursor.close()
+    con.commit()
+
+
+def read_task(con: sqlite3.Connection):
+    cursor = con.cursor()
+    cursor.execute(f'''
+        SELECT * FROM {MY_TODO_TABLE}
```

Comment:
```suggestion
        SELECT id, title, content, done FROM {MY_TODO_TABLE}
```

- 추후 테이블에 컬럼이 더 추가됐을 때
- 테이블의 컬럼 정의 순서가 변경됐을 때

42L에 넘어가는 tuple의 데이터가 예상치 못하게 변경될 수 있어서, 명시적으로 하는 것이 좋아 보여요.